allauth/socialaccount/providers/oauth/views.py

```diff
@@ -76,6 +78,7 @@ def login(self, request, *args, **kwargs):
         try:
             return client.get_redirect(auth_url, auth_params)
         except OAuthError as e:
+            logger.exception('Error processing OAuth')
```

Comment:
**Issue**:  From a logging perspective, these errors can occur in production and don't necessarily indicate exceptional situations that need direct attention. Hence, I think we need to use the regular `error` level.

```suggestion
            logger.error("OAuth authentication error", exc_info=True)
```